exo-go/features/help.feature

```diff
@@ -0,0 +1,43 @@
+Feature: help command
+
+  As a developer not familiar with how to call "exo clean" correctly
+  I want to be given helpful information how to use this application
+  So that I can do the things I intended without having to look this up somewhere else.
+
+
+  Scenario: the user enters 'exo help'
+    When running "exo help" in the terminal
+    Then I see:
+      """
+      Usage:
+        exo [command]
+
+      Available Commands:
+        clean       Removes dangling Docker images and volumes.
+        create      Create a new Exosphere application
+        version     Exosphere go version number
+
+      Use "exo [command] --help" for more information about a command.
```

Comment:
This was generated by the cobra library. I'd prefer we just use whats builtin to cobra instead of implementing our own. Once we switch over to fully using this binary this will be a breaking change